trace-forward/src/Trace/Forward/Utils.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Trace.Forward.Utils
+  ( runActionInLoop
+  ) where
+
+import           Control.Exception (SomeAsyncException (..), fromException, tryJust)
+import           Control.Tracer (showTracing, stdoutTracer, traceWith)
+import           System.Time.Extra (sleep)
+
+import           Trace.Forward.Configuration (HowToConnect)
+
+-- | Run monadic action in a loop. If there's an exception, it will re-run
+--   the action again, after pause that grows.
+runActionInLoop
+  :: IO ()
+  -> HowToConnect
+  -> Word
+  -> IO ()
+runActionInLoop action endpoint prevDelay =
+  tryJust excludeAsyncExceptions action >>= \case
+    Left e -> do
+      logTrace $ "trace-forward, connection with " <> show endpoint <> " failed: " <> show e
+      let currentDelay = mkDelayInSecs
+      sleep $ fromIntegral currentDelay
```

Comment:
In other parts of the code you're using `threadDelay`.  I think it's better to be consistent and use it all the way through.  This makes things harder to confuse seconds vs miliseconds. From which package does `sleep` comes from?

Reply:
From `extra` package.

Reply:
So yes, I could use `sleep` everywhere.

Reply:
Done.

Reply:
Now `sleep` is everywhere.